### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by temerson**
It makes the pretty. The oh-so-pretty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous" />
+    <link rel="stylesheet" href="./Main.css" />
+  </head>
+  <body>
+    <!-- Your Elm application  -->
+    <script src="main.js"></script>
+    <script type="text/javascript">Elm.Main.fullscreen()</script>
```
Comment (mikemedina) [inline index.html:15]: Could these script tags be combined into one?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Journal
+
+Run with []().
```
Comment (mikemedina) [inline README.md]: wut

File: index.html (hunk 1 above)
Comment (temerson) [reply index.html:15]: I tested that and it doesn't seem to work.

**Review by mikemedina — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (temerson) [reply README.md]: I goofed

**Pull request merged**